code/modules/surgery/organs/organ_external.dm

```diff
@@ -105,12 +102,23 @@
 
 /obj/item/organ/external/New(mob/living/carbon/holder)
 	..()
-	var/mob/living/carbon/human/H = holder
-	icobase = dna.species.icobase
-	if(istype(H))
-		replaced(H)
-		sync_colour_to_human(H)
-	get_icon()
+	if(holder)
+		if(ishuman(holder))
+			var/mob/living/carbon/human/H = holder
+			icobase = H.dna.species.icobase
+			if(istype(H))
+				replaced(H)
+				sync_colour_to_human(H)
+			get_icon()
```

Comment:
`get_icon()` needs to always be called, otherwise the limb won't have any sprites if it's spawned in separately.

Also since this doesn't really do anything if `holder` isn't a human, you should be able to compress it a bit to this:
```dm
/obj/item/organ/external/New(mob/living/carbon/holder)
	..()
	if(ishuman(holder))
		var/mob/living/carbon/human/H = holder
		icobase = H.dna.species.icobase
		replaced(H)
		sync_colour_to_human(H)
	get_icon()
```